sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -259,6 +304,13 @@ struct sub_group_mask {
 
   static constexpr size_t max_bits = /* implementation-defined */;
 
+  sub_group_mask();
```

Comment:
It would be nice to add a comment here too saying that these are available starting in revision 2.

Reply:
Added corresponding note in 14c468290d540162908c8331758ab1db9cc57324